### PR TITLE
GDB-8013 Add additional properties field for ontop repositories

### PR DESCRIPTION
**Pull request by yordanalexandrov**
## What
When creating ontop repository the user should be able to pass additional ontop properties

## Why
Ontop supports properties that are not exposed in the workbench and the used has to go and manually configure them in the ontop.properties file

## How
- added a text area when creating the ontop repository where the user can add additional properties in properties file format
- Added cypress test
- unskipped cypress test for `should create an Ontop repository` as the waited MR was merged.
- updated `form_request` command to current cypress version

### CODE REVIEW

File: src/js/angular/repositories/controllers.js [hunk 1]
```diff
@@ -661,15 +662,14 @@ function AddRepositoryCtrl($scope, toastr, $repositories, $location, $timeout, U
         RepositoriesRestService.createRepository($scope.repositoryInfo)
             .then(function () {
                 toastr.success($translate.instant('created.repo.success.msg', {repoId: $scope.repositoryInfo.id}));
-                $repositories.init();
+                return $repositories.init().then(() => $scope.goBackToPreviousLocation());
```
Comment (svilenvelikov) [inline src/js/angular/repositories/controllers.js:665]: flatten this promise chain pls

File: test-cypress/integration/repository/repositories.spec.js [hunk 2]
```diff
@@ -382,131 +382,120 @@ describe('Repositories', () => {
         getConstraintUploadButton().scrollIntoView().should('be.visible').and('not.be.disabled');
     });
 
-    // Remove skip, when https://gitlab.ontotext.com/graphdb-team/graphdb/-/merge_requests/1584 is merged
     //Create Ontop repository and test ontop functionality
-    it.skip('should create an Ontop repository', () => {
+    it('should create an Ontop repository', () => {
```
Comment (svilenvelikov) [inline test-cypress/integration/repository/repositories.spec.js:386]: This is a hideous test

**Review by svilenvelikov — APPROVED** [review]

File: src/js/angular/repositories/controllers.js (hunk 1 above)
Comment (yordanalexandrov) [reply src/js/angular/repositories/controllers.js]: done

File: test-cypress/integration/repository/repositories.spec.js (hunk 2 above)
Comment (yordanalexandrov) [reply test-cypress/integration/repository/repositories.spec.js:387]: well ...

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**